src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,3 @@
+$grey-star-color: #f3f3f3;
+$standart-inline-pad: 16px;
+$standart-black-color: #060b35;
```

Comment:
The variable for standard black color is correctly defined. Ensure all color values used in your styles are defined here.